src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -74,9 +74,18 @@ internal sealed class MeterProviderSdk : MeterProvider
                     this.reader = new CompositeMetricReader(new[] { this.reader, reader });
                 }
 
-                if (reader is BaseExportingMetricReader baseExportingMetricReader)
+                if (reader is PeriodicExportingMetricReader periodicExportingMetricReader)
+                {
+                    exportersAdded.Append(((PeriodicExportingMetricReader)reader).Exporter);
+                    exportersAdded.Append(" (Paired with PeriodicExportingMetricReader exporting at ");
+                    exportersAdded.Append(((PeriodicExportingMetricReader)reader).ExportIntervalMilliseconds);
```

Comment:
no special casing for Timeout.Infinite, but i think even without that, this is still a useful log message.